python/python/bystro/ancestry/train.py

```
@@ -361,35 +361,37 @@
 def _load_ancestry_df() -> pd.DataFrame:
-    ancestry_df = pd.read_csv(ANCESTRY_INFO_PATH, sep="\t", index_col=0)
-    assert_equals("number of rows", 3500, "actual number of rows", len(ancestry_df))
+    ancestry_df = pd.read_csv(ANCESTRY_INFO_PATH, sep=",")
+    assert_equals("number of rows", 3195, "actual number of rows", len(ancestry_df))
     expected_samples = ["NA12865", "HG03930", "NA19171"]
     assert_true(
-        "Index passes spot checks", all(sample in ancestry_df.index for sample in expected_samples)
+        "Sample name column passes spot checks", 
+        all(sample in ancestry_df['Sample name'].values for sample in expected_samples)
     )
+    ancestry_df.set_index('Sample name', inplace=True)
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)